File: Antidote/PortraitNavigationController.m
//
//  PortraitViewController.m
//  Antidote
//
//  Created by Nicholas Palamar on 4/15/15.
//  Copyright (c) 2015 dvor. All rights reserved.
//

#import "PortraitNavigationController.h"

@implementation PortraitNavigationController

- (BOOL)shouldAutorotate
{
    return NO;
}

- (NSUInteger)supportedInterfaceOrientations
{
    return UIInterfaceOrientationMaskPortrait;
}

- (UIInterfaceOrientation)preferredInterfaceOrientationForPresentation
{
    return UIInterfaceOrientationPortrait;
}

@end

File: Antidote/PortraitNavigationController.swift
//
//  PortraitNavigationController.swift
//  Antidote
//
//  Created by Dmytro Vorobiov on 13/10/15.
//  Copyright © 2015 dvor. All rights reserved.
//

import UIKit

class PortraitNavigationController: UINavigationController {
    override func shouldAutorotate() -> Bool {
        return false
    }

    override func supportedInterfaceOrientations() -> UIInterfaceOrientationMask {
        return UIInterfaceOrientationMask.Portrait
    }
}
